web/app/services/auth.service.ts

```diff
@@ -6,14 +6,14 @@ import {shareReplay, tap} from 'rxjs/operators';
 import {LocalStorageKeys} from '../common/constants';
 
 // Auth server is currently locally hosted.
-const HOSTNAME = '/data';
+const HOSTNAME = '/api';
```

Comment:
`HOSTNAME` variable probably needs to be renamed to something like `API_ROOT`, right?

Reply:
that's a good point. HOSTNAME in the future, i'm envisioning being a config var you can adjust when spinning the server up.